cogs/torrents.py

```diff
@@ -0,0 +1,66 @@
+import os
+import discord
+import requests
+import json
+from discord.ext import commands
+
+
+def load_json(token):
+    with open('./config.json') as f:
+        config = json.load(f)
+    return config.get(token)
+
+
+def setup(client):
+    client.add_cog(Torrents(client))
+
+
+def get_magnet(search: str) -> discord.Embed:
+    """Get a magnet link for a torrent"""
+
+    failure = discord.Embed(
+        title='Unable to find torrent',
+        url=f'https://www.thepiratebay.org/search/{search}',
+        description='No Magnet Found'
+    )
+    failure.set_footer(text='Better luck next time!')
+
+    try:
+        response = requests.get(f'https://apibay.org/q.php?q={search}')
+    except Exception:
+        return failure
+
+    # Top torrent
+    target = json.loads(response.content)[0]
+
+    # No torrent found
+    if target.get('id') == '0':
+        return failure
+
+    # Top torrent magnet link
+    desc = f"Magnet: magnet:?xt=urn:btih:{target.get('info_hash')}&dn={target.get('name')}\n\n"
+    desc += f"Seeders: {target.get('seeders')}\nLeachers: {target.get('leechers')}\n"
+    desc += f"Uploader: {target.get('username')}\nSize:  {target.get('size')}"
+
+    success = discord.Embed(
+        title=target.get('name'),
+        description=desc
+    )
+    success.set_footer(text='I am not responsible for anything downloaded via this cog')
```

Comment:
change cog to command.
Does it even need this as a footer tho?

Reply:
Probably not. Removed.